docs/run-taco.md

```diff
@@ -6,8 +6,15 @@ Note: the below is intended for connector developers. Customer-facing documentat
 
 # Load order and class name collisions
 
+## Tableau 2022.2 and above
+If a connector has the same class as a connector that has already been registered, the new connector will be rejected. This means that connectors loaded first have precedence when two connectors share the same class nameTableau will load the connector with the highest connector version number (the plugin-version attribute in the connector manifest).
```

Comment:
I don't follow this sentence. Also add space in nameTableau

Reply:
Copy/paste error I think, I can't figure it out either.